lib/utils/cli.go

```diff
@@ -522,3 +523,21 @@ type PredicateError struct {
 func (p PredicateError) Error() string {
 	return fmt.Sprintf("%s\nCheck syntax at https://goteleport.com/docs/setup/reference/predicate-language/#resource-filtering", p.Err.Error())
 }
+
+func FormatAlertOutput(alert types.ClusterAlert) string {
```

Comment:
Nit:
```suggestion
func FormatAlert(alert types.ClusterAlert) string {
```

Also, add godoc.